docs/general/glossary.md

```diff
@@ -464,14 +464,13 @@ developers to create {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama
 
 ## ParaID
 
-A unique numeric (non-negative integer) identifier for a parathread.
+A unique numeric (non-negative integer) identifier for a parachain.
 
 ## Parathread
 
-A blockchain that works within the confines of the
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} Host, thus allowing it to connect to
-the Polkadot Relay Chain. This can be done either by becoming a parachain or connecting on a "pay as
-you go" basis. All parachains start their life-cycle as a parathread.
+A parathread can have two meanings: a substrate-based chain not yet parachain (i.e. not connected to
+a {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} core), or a on-demand parachain
+(i.e. a parachain using instantaneous coretime).
```

Comment:
https://wiki.polkadot.network/docs/learn-agile-coretime#is-it-on-demand-coretime-or-instantaneous-coretime 
Avoid referring to "instantaneous coretime"

```suggestion
a {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} core), or as an on-demand parachain
(i.e. a parachain using on-demand coretime).
```